web/themes/custom/nysenate_theme/src/patterns/components/nysenate-happening-now/nysenate-happening-now.twig

```diff
@@ -1,30 +1,39 @@
-{{ attach_library('nysenate_theme/nysenate-happening-now') }}
+{{ attach_library('nysenate_theme/featured-story') }}
```

Comment:
Why did the library change?

Reply:
because they were reinventing the wheel with a new css file, when all the styles and html should have been the same as a featured story ... their html wasn't working or complete

Reply:
Maybe  a naming issue, then.  Something to think about for later, I guess.. libraries should be named appropriately to their use cases, and be limited to that use case.